packages/common/src/localStorage/keys.ts

```diff
@@ -20,6 +20,7 @@ export type LocalStorageRecord = {
   '/theme/logo': string;
   '/mru/credentials': AuthenticationCredentials;
   '/authentication/token': string;
+  '/authentication/storeid': string;
```

Comment:
would like to lift the auth aspects out to a cookie - mainly to set expiry, but also for a separation between the rest of the local storage concerns.

Reply:
I'm not following this point, might have to get you to spell it out to me 😁  🤔